lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -281,7 +282,11 @@ def parse_rates_response(response, origin, destination)
       rates = doc.root.xpath('price-quote').map do |node|
         service_name  = node.at("service-name").text
         service_code  = node.at("service-code").text
-        total_price   = node.at('price-details/due').text
+        total_price   = if @rates_exclude_tax
```

Comment:
You can use string template to do it in a single line and still readable
`total_price = "price-details/#{@rates_exclude_tax ? 'base' : 'due' }"`